evennia/server/portal/webclient_ajax.py

```diff
@@ -171,19 +183,19 @@ def lineSend(self, csessid, data):
             dataentries.append(jsonify(data))
             self.databuffer[csessid] = dataentries
 
-    def client_disconnect(self, csessid):
+    def client_disconnect(self, cusessid):
```

Comment:
The docstring still mentions `csessid`. It should reflect that cusessid is a combination of the session id and browser id.